tests/fixed/Exceptions.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Exceptions;
```

Comment:
Are these files supposed to have no body? 🤔

Reply:
Yup, it's a base file and the body is added via 80 and 81 patches. No support in < 8

Reply:
<sub>let's just drop php 7</sub>

Reply:
Agree but not my call. Let's have @Doctrine team do that.

Reply:
Honestly why not? We run the cs jobs on PHP 8, and can use `^9.0 || ^10.0` as a version constraint on projects that need to support PHP 7.

Reply:
I'm happy to drop php 7 support if you say it's ok with you. Can do it in another PR.

Reply:
Ah but I get why you would be reluctant to do so: https://github.com/doctrine/coding-standard/pull/226

@beberlei , if the code still works, but the tests are harder to write, can we maybe drop support for PHP 7?

Reply:
This repository does not contain any code, so I don't see a point in requiring a higher PHP version than the sum of our dependencies. @greg0ire is right that we don't run PHPCS itself on PHP 7 anymore, but we run it on code that needs to run on PHP 7.

What the test suite currently covers isn't running the ruleset *on different PHP versions*, but running it *for different language levels*. And we would still need that matrix, even if we bumped the min PHP version in composer.json to 8.2.

I understand that reviewing those patch files is cumbersome. Maybe it would be a little less painful if we created dedicated `fixed` folders per language level?

Reply:
> Maybe it would be a little less painful if we created dedicated fixed folders per language level?

Inputs are also different for different PHP versions.

Reply:
Maintenance would still be PIA (since maintaining eoled phps sucks and there's too many versions already) but it may facilitate reviews, yes.